airbyte-integrations/connectors/source-amazon-seller-partner/unit_tests/integration/utils.py

```diff
@@ -11,7 +11,7 @@
 from airbyte_cdk.test.catalog_builder import CatalogBuilder
 from airbyte_cdk.test.entrypoint_wrapper import EntrypointOutput, read
 from airbyte_cdk.test.mock_http import HttpMocker
-from airbyte_cdk.test.mock_http.response_builder import _get_unit_test_folder
+from airbyte_cdk.test.mock_http.response_builder import get_unit_test_folder
```

Comment:
Oops, this is unfortunate! We will need to update the CDK version before doing that. [It uses CDK version 0.90.0](https://github.com/airbytehq/airbyte/blob/eb9168cf9e87f37a28e9acfab2ab47eabda4bdec/airbyte-integrations/connectors/source-amazon-seller-partner/poetry.lock#L5) which means a lot of breaking changes. 

The options I see are:
* Pay the debt of importing private methods now by updating the CDK (this seems to increase the scope way too much)
* Keep `_get_unit_test_folder` and have it just call `get_unit_test_folder` internally with a big red comment in the CDK and `source-amazon-seller-partner` that says why we have this and when we can clean this. In that case, if you want to move `get_unit_test_folder` to `data`, that works too

I prefer the second option personally but will let you choose.

Reply:
OK, going with the second option.